mempool/src/proto/mempool.proto

```diff
@@ -57,7 +57,7 @@ message AddTransactionWithValidationResponse {
   // transaction will have version bigger than this 'current_version'
   uint64 current_version = 1;
   // The result of the transaction submission
-  MempoolAddTransactionStatus status = 2;
+  mempool_status.MempoolAddTransactionStatus status = 2;
```

Comment:
can we get rid of mempool_status and embed its content here? It's only one message and one enum

Reply:
I think there's some reason for keeping them separate, but i'll leave that to the mempool folks to decided what's best to do there